### ISSUE TITLE
Warning: Calling Formula.sha1 is deprecated!

### ISSUE THREAD

**Issue by stanislaw** [AlexDenisov/homebrew-xcconf#1]
```
Warning: Calling Formula.sha1 is deprecated!
Use Formula.sha256 instead.
/usr/local/Library/Taps/alexdenisov/homebrew-xcconf/xcconf.rb:4:in `<class:Xcconf>'
Please report this to the alexdenisov/xcconf tap!
```

**Comment by phil-r** [comment]
Now it's actually an error:

```
Error: Calling Formula.sha1 is disabled!
Use Formula.sha256 instead.
/usr/local/Homebrew/Library/Taps/alexdenisov/homebrew-xcconf/xcconf.rb:4:in `<class:Xcconf>'
Please report this to the alexdenisov/xcconf tap!
```

**Comment by AlexDenisov** [comment]
Solved via #2
